src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
```

Comment:
You should try destructuring of `action` in `for..of` loop

Reply:
> You should try destructuring of `action` in `for..of` loop

Yeah, but depending on `type` we need to use a different property. If type is 'addProperties' we use `extraData` and if 'removeProperties' - `keysToRemove`. So if we use destructuring we always have extra variable.

Reply:
agreed!